pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1728,7 +1733,7 @@ def on_encrypt_toggled(self, encrypted):
             encrypted.get_active() and encrypted.get_sensitive()
         )
 
-    def _populate_container(self, device):
+    def _populate_container(self):
```

Comment:
You probably know and pylint will tell you, and it is WIP with more pylint issues, but the device is used at the bottom of the method.

Reply:
There were some unfinished parts of the custom spoke. They should be fixed now.